LibGUI/GWindow.cpp

```diff
@@ -262,14 +264,54 @@ void GWindow::event(CEvent& event)
     }
 
     if (event.type() == GEvent::KeyUp || event.type() == GEvent::KeyDown) {
-        if (m_focused_widget)
-            return m_focused_widget->event(event);
-        if (m_main_widget)
-            return m_main_widget->event(event);
+        auto keyevent = static_cast<GKeyEvent&>(event);
+        if (keyevent.logo() && event.type() == GEvent::KeyUp) {
+            if (m_keybind_mode) {
+                m_keybind_mode = false;
+            } else {
+                m_keybind_mode = true;
+                find_keyboard_selectable();
+                m_entered_keybind = "";
+            }
+            update();
+            return;
+        }
+
+        if (m_keybind_mode) {
+            if (event.type() == GEvent::KeyUp) {
+                StringBuilder builder;
+                builder.append(m_entered_keybind);
+                builder.append(keyevent.text());
+                m_entered_keybind = builder.to_string();
+
+                auto found_widget = m_hashed_potential_keybind_widgets.find(m_entered_keybind);
+                if (found_widget != m_hashed_potential_keybind_widgets.end()) {
+                    m_keybind_mode = false;
+                    const auto &point = Point();
```

Comment:
No need for a named local variable here, just pass `{}` instead of `point` to the `GMouseEvent` constructors.

Reply:
That doesn't compile though.

Reply:
Ah right because of forwarding, got it. You can just use `Point()` then :)